etc/meson.build

```diff
@@ -39,3 +39,6 @@ configure_file(
 )
 
 nox_inc = include_directories('.')
+
+wireshark = dependency('wireshark')
```

Comment:
Personally I'd add a `meson.build` inside the `dissectors` directory, then have this dep inside there.

I'd also add `required: true` to it